onecgiar-pr-server/src/api/versioning/versioning.repository.ts

```diff
@@ -84,25 +84,32 @@ export class VersionRepository extends Repository<Version> {
       });
   }
 
-  $_getAllInovationDevToReplicate(
+  async $_getAllInovationDevToReplicate(
```

Comment:
May I know why this function is async? There is no need for it, as we are returning a promise...